hw/ip/i2c/doc/_index.md

```diff
@@ -246,7 +246,10 @@ As described in the I2C specification, a target device can pause a transaction b
 A target device stretches the clock automatically when a host device reads from the target and the target's TX FIFO is empty.
 Also, a target device stretches the clock automatically when a host device writes to the target and the target's ACQ FIFO is full.
 The present implementation also supports few additional modes in which a target device can stretch the clock after completing a transaction if it needs more time, e.g., to store the acquired data byte in flash memory, etc.
-If {{< regref STRETCH_CTRL.ENABLEADDR >}} is asserted, a target stretches the clock after acquiring an address and making certain that the transaction is intended for it and not for another device on the bus.
+If {{< regref STRETCH_CTRL.EN_ADDR_TX >}} or {{< regref STRETCH_CTRL.EN_ADDR_ACQ >}} is asserted, a target stretches the clock after acquiring an address and making certain that the transmit (read by a host) or acquire (write by a host) transaction, respectively, is intended for it and not for another device on the bus.
+A target does not stretch the clock, if a corresponding {{< regref STRETCH_CTRL.STOP_TX >}} or {{< regref STRETCH_CTRL.STOP_ACQ >}} bit is asserted as well.
```

Comment:
This sentence took me a while to parse. Is the rephrasing below accurate?
```suggestion
However, if either the {{< regref STRETCH_CTRL.STOP_TX >}} or {{< regref STRETCH_CTRL.STOP_ACQ >}} bit is asserted, a target does not stretch the clock
```

Reply:
What I meant to say here is the following:
A target does not stretch the clock if {{< regref STRETCH_CTRL.STOP_TX >}} is asserted even when {{< regref STRETCH_CTRL.EN_ADDR_TX >}} is asserted.
Similarly, a target does not stretch the clock if {{< regref STRETCH_CTRL.STOP_ACQ >}} is asserted even when {{< regref STRETCH_CTRL.EN_ADDR_ACQ >}} is asserted.
The bits EN_ADDR and STOP have to be paired:
A target will stretch the clock if {{< regref STRETCH_CTRL.STOP_TX >}} is asserted when {{< regref STRETCH_CTRL.EN_ADDR_ACQ >}} is asserted.
Also, a target will stretch the clock if {{< regref STRETCH_CTRL.STOP_ACQ >}} is asserted when {{< regref STRETCH_CTRL.EN_ADDR_TX >}} is asserted.

Reply:
@igk8 i feel like it might be clearer to say, "to resume normal operation and stop clock stretching, set STOP_TX / STOP_ACQ".

I think that's how you described it in the register fields, and it feels clearer to me. Just my opinion though, I don't have all the context.

Reply:
@martin-lueker @tjaychen I've modified this sentence, making it clearer hopefully. Please review.